plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -1043,13 +1044,15 @@ public void testIoExplain()
         computeActual("CREATE TABLE test_io_explain WITH (partitioned_by = ARRAY['orderkey', 'processing']) AS SELECT custkey, orderkey, orderstatus = 'P' processing FROM orders WHERE orderkey < 3");
 
         EstimatedStatsAndCost estimate = new EstimatedStatsAndCost(2.0, 40.0, 40.0, 0.0, 0.0);
+        boolean withoutData = estimate.getOutputRowCount() == 0d ? true : false;
```

Comment:
We now the expected result (constraint here shouldn't be none)
we don't need a variable and conditional initialization for that